src/components/about/Landing.jsx

```diff
@@ -5,19 +5,19 @@ const Landing = () => {
   return (
     <div className="w-full flex justify-center items-center h-[70vh]">
       <div className="flex-row w-auto">
-        <div className="h-auto flex gap-8 ">
-          <div className="flex w-1/2 justify-end items-center">
+        <div className="h-auto flex sm:flex-row flex-col gap-8 items-center">
+          <div className="flex w-1/2 justify-end items-center mr-2 sm:m-0">
             <Image src={chart} className="h-full w-auto m-0" alt="chart" />
           </div>
           <div className="px-2 flex w-full justify-center items-start">
             <div className="flex flex-col w-3/4 lg:w-auto">
-              <p className="text-5xl py-1.5 font-bold text-transparent bg-clip-text bg-gradient-to-r from-hiss-blue to-hiss-purple">
+              <p className="text-5xl text-center py-1.5 font-bold text-transparent bg-clip-text bg-gradient-to-r from-hiss-blue to-hiss-purple">
                 Highlander Statistics Society
               </p>
-              <p className="mt-0 my-0 text-white px-0 py-2.5 font-bold text-center text-3xl h-14 w-4/5 bg-gradient-to-r from-hiss-blue to-hiss-purple rounded-full">
+              <p className="sm:m-0 mr-24 text-white px-0 py-2.5 font-bold text-center sm:text-3xl sm:h-14 sm:w-4/5 w-80 bg-gradient-to-r from-hiss-blue to-hiss-purple rounded-full">
                 at Unversity of California Riverside
               </p>
-              <p className="my-1 py-1 h-10 w-6/12">
+              <p className="sm:my-1 py-1 mb-10 h-10 sm:w-6/12 w-full">
```

Comment:
same thing with the mb-10

Is you screenshot showing on desktop or laptop?

Reply:
showing on desktop monitor